src/main/java/frc/trigon/robot/commands/CommandConstants.java

```diff
@@ -24,16 +27,22 @@ public class CommandConstants {
     private static final double
             MINIMUM_TRANSLATION_SHIFT_POWER = 0.18,
             MINIMUM_ROTATION_SHIFT_POWER = 0.3;
+    private static final double ROTATION_JOYSTICK_DEADBAND = 0.4;
```

Comment:
Go with how you called it in other places: `JOYSTICK_ORIENTED_ROTATION_DEADBAND`